src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
+            strykerRunnerMock.VerifyAll();
 
-            var filePatterns = actualOptions.FilePatterns.ToArray();
-            filePatterns.Count(x => x.IsExclude).ShouldBe(3);
-            filePatterns.ShouldContain(firstFileToExclude);
-            filePatterns.ShouldContain(secondFileToExclude);
-            filePatterns.ShouldContain(thirdFileToExclude);
+            inputs.ReportersInput.SuppliedInput.ShouldContain(Reporter.Html.ToString());
+            inputs.ReportersInput.SuppliedInput.ShouldContain(Reporter.Dots.ToString());
         }
 
         [Theory]
-        [InlineData("--mutate")]
-        [InlineData("-m")]
-        public void StrykerCLI_WithFilePatternSet_ShouldPassFilePatternSetToStryker(string argName)
+        [InlineData("--project")]
+        [InlineData("-p")]
+        public void ShouldPassProjectArgumentsToStryker_WithProjectArgument(string argName)
         {
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            StrykerOptions actualOptions = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.1);
+            target.Run(new string[] { argName, "SomeProjectName.csproj" });
 
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => actualOptions = c)
-                .Returns(runResults)
-                .Verifiable();
+            strykerRunnerMock.VerifyAll();
 
-            var target = new StrykerCLI(mock.Object);
+            inputs.ProjectUnderTestNameInput.SuppliedInput.ShouldBe("SomeProjectName.csproj");
+        }
 
-            target.Run(new[] { argName, @"['**/*Service.cs','!**/MySpecialService.cs', '**/MyOtherService.cs{1..10}{32..45}']" });
+        [Theory]
+        [InlineData("--solution")]
+        [InlineData("-s")]
+        public void ShouldPassSolutionArgumentPlusBasePathToStryker_WithSolutionArgument(string argName)
+        {
+            target.Run(new string[] { argName, "SomeSolutionPath.sln" });
 
-            var firstFileToExclude = FilePattern.Parse("**/*Service.cs");
-            var secondFileToExclude = FilePattern.Parse("!**/MySpecialService.cs");
-            var thirdFileToExclude = FilePattern.Parse("**/MyOtherService.cs{1..10}{32..45}");
+            strykerRunnerMock.VerifyAll();
 
-            var filePatterns = actualOptions.FilePatterns.ToArray();
-            filePatterns.Length.ShouldBe(3);
-            filePatterns.ShouldContain(firstFileToExclude);
-            filePatterns.ShouldContain(secondFileToExclude);
-            filePatterns.ShouldContain(thirdFileToExclude);
+            inputs.SolutionPathInput.SuppliedInput.ShouldBe("SomeSolutionPath.sln");
         }
 
         [Theory]
-        [InlineData("--diff")]
-        [InlineData("-diff")]
-        public void ShouldEnableDiffFeatureWhenPassed(string argName)
+        [InlineData("--verbosity")]
+        [InlineData("-V")]
```

Comment:
Having both `-v` and `-V` is quite subtle imo, I would prefer either removing short versions then or finding alternatives (although not hard to do in this particular case)

Reply:
My thinking was you almost never need to change the verbosity of the commandline output anyway so it's fine as uppercase shorthand even though it might be slightly confusing. While I agree that it's subtle it's also not uncommon in other clis. If we don't go with -V I would rather not have a shorthand at all. One of the things I personally dislike the most in clis is when the shorthand does not match the long version at all. Do you have a suggestion that would fit as a shorthand?

Reply:
Well maybe you could have -v here and -ver for version? But this applies also to other cases, like -l and -L and so on.

Reply:
The same argument applies there. log to file should not be an often used option so I felt that it was safe to make it `-L`. If this causes too much confusion I would be happy to remove the shorthand for both as they don't really need a shorthand anyway. Would that be acceptable or would you prefer shorthands for those as well? I don't want long-shorthands so I won't do more than one character because at that point you might as well just type it out. We currently have options such as `--diff` with the shorthand `-diff` 🙄

Reply:
I am then up for just removing the shorthands for these options. Indeed, shortcuts are mostly useful with longer options' names (like `max-concurrent-test-runners`), which is not the case here. 

Btw, -diff vs --diff looks odd, I was confused with that when reading docs :D

Reply:
I'll remove these confusing shorthands 👍